src/components/[guild]/JoinModal/hooks/useConnectPlatform.ts

```diff
@@ -28,14 +28,40 @@ const useConnectPlatform = (
   onSuccess?: () => void,
   isReauth?: boolean // Temporary, once /connect works without it, we can remove this
 ) => {
-  const { addDatadogAction, addDatadogError } = useDatadog()
-  const showErrorToast = useShowErrorToast()
-
-  const { mutate: mutateUser, platformUsers } = useUser()
+  const { platformUsers } = useUser()
   const { onOpen, authData, isAuthenticating, ...rest } =
     platformAuthHooks[platform]?.() ?? {}
   const prevAuthData = usePrevious(authData)
 
+  const { onSubmit, isLoading, response } = useConnect(onSuccess)
+
+  useEffect(() => {
+    // couldn't prevent spamming requests without all these three conditions
+    if (!platformUsers || !authData || prevAuthData) return
+    // const alreadyConnected = platformUsers.some(
```

Comment:
I think we could delete these lines, they're commented since months

Reply:
Removed it in https://github.com/agoraxyz/guild.xyz/pull/733/commits/405621046331f229e6a122e75b7e0d40373b4c72 👍